client/components/editor/ActivityContent/ContainerList.vue

```diff
@@ -54,7 +54,8 @@ export default {
   inheritAttrs: false,
   inject: ['$ccRegistry'],
   props: {
-    containerGroup: { type: Array, default() { return []; } },
+    containerGroup: { type: Array, default: () => ({}) },
+    commentsWithinElements: { type: Object, required: true },
```

Comment:
Since there will be other things included use something more generic like processedElements